src/subscriber/Subscriber.ts

```diff
@@ -82,11 +82,16 @@ export class Subscriber extends OTEventEmitter<{
 
     const onParticipantUpdated = (dailyEvent?: DailyEventObjectParticipant) => {
       if (!dailyEvent) return;
-      // Create stream and add it to subscriber
 
-      if (this.id === dailyEvent.participant.session_id) {
+      // Create stream and add it to subscriber if it does not exist
```

Comment:
The previous line was working by accident. `this.id` is the id of the HTML DOM element, _not_ the session_id.